src/Culling/babylon.boundingBox.ts

```diff
@@ -182,7 +206,7 @@
          * @returns true if there is an intersection
          */
         public isInFrustum(frustumPlanes: Plane[]): boolean {
-            return BoundingBox.IsInFrustum(this.vectorsWorld, frustumPlanes);
+            return BoundingBox.IsInFrustum(this.vectorsWorldToRef(Tmp.Vector3), frustumPlanes);

```

Comment:
Impact on performance here and below should be negligible as we just add a few assignements.

Reply:
This is something I would like to get an evaluation because we trade 16 vector3 vs 24 assignements per submesh per frame. (this will start to be a thing if you have 4000 meshes)

I do not want to favor memory against perf.

I would prefer to have a way to use one or the other depending on user needs.

Reply:
Got your point, that's something that would need to be measured. before merging. We might  consider having a single array of 24 number instead of 8 Vector3 to speed up things a bit more for such functions that are called every frame.